jackett.py

```diff
@@ -24,6 +26,15 @@
     from urllib import request as urllib_request
     from urllib.error import HTTPError as HTTP_Error
 
+try:
+    with open(os.path.join(os.path.dirname(os.path.realpath(__file__)), config_file)) as fd:
+        config_data = json.load(fd)
```

Comment:
yaml seems like a more user friendly format, what do you think?

Reply:
or ini, since python has a standard lib for it

https://docs.python.org/3/library/configparser.html

because I dont think json has comments, where you can explain users what do type where